discord/guild.py

```diff
@@ -1063,7 +1065,7 @@ def _create_channel(
         self,
         name: str,
         channel_type: Literal[ChannelType.text],
-        overwrites: Dict[Union[Role, Member], PermissionOverwrite] = ...,
+        overwrites: Mapping[Union[Role, Member], PermissionOverwrite] = ...,
```

Comment:
I'm not sure How I feel about these `Mapping` replacements.
We used to do a `isinstance(x, dict)` check which had to be modified to support it.

Undo?

Reply:
I can't readily imagine someone using a non-dict `Mapping` being used here.